release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -768,6 +768,37 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
+
+* The console used a prior version of the `KafkaSource` object that used the `resources` and `service account` fields in their specification. The latest v1beta1 version of the `KafkaSource` object removed these fields, due to which the user was unable to create the `KafkaSource` object with the v1beta1 version. This issue has been fixed now and the user is able to create the `KafkaSource` object with the v1beta1 version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892653[*BZ#1892653*])
+
+* Previously, when you created an application using source code from Git repositories with the `.git` suffix, and then clicked the edit source code link, a "page not found" error was displayed. This fix removes the `.git` suffix from the repository URL and transforms the SSH URL to an HTTPS URL. The generated link now leads to the correct repository page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896296[*BZ#1896296*])
+
+* Previously, the underlying `SinkBinding` resources were shown in the *Topology* view, along with the actual source created in the case of `Container Source` and `KameletBinding` resources, confusing users. This issue was fixed. Now, only the actual resource created for the event source is displayed in the *Topology* view, and the underlying `SinkBinding` resources, if created, are displayed in the sidebar. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906685[*BZ#1906685*])
+
+* Previously, when you installed the Serverless Operator, without creating the eventing custom resource, a channel card was displayed. When you clicked the card, a confusing alert message was displayed. This issue has now been fixed. The channel card, with a proper alert message, is now displayed only if the channel custom resource definition is present. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909092[*BZ#1909092*])
+
+* Previously, when you closed the web terminal connection, all the terminal output from that session, which can be useful to you, disappeared. This issue has been fixed. The terminal output is now retained even after the session is closed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1909067[*BZ#1909067*])
+
+* Technology preview badges were displayed on the Eventing user interface although it had its GA release with {product-title} 4.6. The Technology preview badges are now removed and the changes were back-ported to the {product-title} 4.6.9 version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894810[*BZ#1894810*])
+
+* Previously, volume mounts for deployments were not preserved if the deployment was edited using the console edit flows. The modified deployment YAML overwrote or removed the volume mounts in the pod template specification. This issue has been fixed. The *volume mounts* are now preserved even when the deployment is edited using the console edit flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1867965[*BZ#1867965*])
```

Comment:
How come just the one final instance of "volume mounts" is marked up with asterisks?

Reply:
I have removed the markup now. Done!